package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internetarchive/bookreader",
-  "version": "5.0.0-10",
+  "version": "5.0.0-12-multiple-files",
```

Comment:
blocker: let's rebase and update with what is in main branch.

the only package.json & package-lock.json changes should be for the multiple-files icon <icon-...> as we are deprecating the use on `<ia-icon>` (discussed during pairing).